cads_adaptors/tools/logger.py

```diff
@@ -0,0 +1,39 @@
+# Taken from gecko/logger.py
```

Comment:
I thought this looked familiar!
Does it work in the worker logs?

Reply:
k9s dos not show the formatting, and maybe B-open have some other formatting set up somewhere. So I think we should just remove the formatting for now and maybe add it in again when we know what we want.

Reply:
I cleaned the file from un-rendered formatting.
There is an environmental variable ADAPTORS_LOG_LEVEL to be set in the definitions of workers at cads-deployment/cads-configuration level. the default value is debug...